modules/account-lib/src/coin/cspr/transferBuilder.ts

```diff
@@ -33,7 +45,9 @@ export class TransferBuilder extends TransactionBuilder {
   initBuilder(tx: Transaction): void {
     super.initBuilder(tx);
     this.transaction.setTransactionType(TransactionType.Send);
-    // TODO(STLX-663): init to and amount
+    this.to(this.getTransferDestinationAddress(tx.casperTx.session));

```

Comment:
Please also change the definition for `to()` to use isValidAddress from './utils' instead